addons/event/static/src/views/event_registration_list_controller.js

```diff
@@ -0,0 +1,43 @@
+import { EventRegistrationSummaryDialog } from "@event/client_action/event_registration_summary_dialog";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { listView } from "@web/views/list/list_view";
+import { ListController } from "@web/views/list/list_controller";
+
+export class EventRegistrationListController extends ListController {
+
+    setup() {
+        super.setup();
+        this.dialog = useService("dialog");
+        this.orm = useService("orm");
+    }
+
+    async openRecord(record) {
+        if (this.props.context.is_registration_desk_view) {
```

Comment:
Same here, can't we locate the code in ``onClickSelectAttendee`` as it is specific to barcode, not list view ?